core/src/models/RecordProperty.ts

```diff
@@ -139,6 +141,30 @@ export class RecordProperty extends CommonModel<RecordProperty> {
     return instance;
   }
 
+  // TODO: I want to figure out the types to move these to CommonModel
+  static async updateAllinBatches(
+    instances: RecordProperty[],
+    values: { [key: string]: any }
+  ) {
+    const ids = instances.map((i) => i.id);
+    return this.updateAllinBatchesById(ids, values);
+  }
+
+  static async updateAllinBatchesById(
+    ids: string[],
+    values: { [key: string]: any }
+  ) {
+    const max = config.batchSize.internalWrite;
+    const queue: string[] = Array.from(ids);
```

Comment:
Down the rabbit hole I go... https://stackoverflow.com/questions/40548213/array-from-vs-spread-syntax